Source/AtomicApp/Player/IPCPlayerApp.cpp

```diff
@@ -206,6 +213,12 @@ namespace Atomic
 
     void IPCPlayerApp::Stop()
     {
+        if (debugPlayer_)
+        {
+            context_->GetSubsystem<JSDebugger>()->AutoReconnect = false;
+            context_->GetSubsystem<JSDebugger>()->Shutdown();
```

Comment:
```Shutdown``` can clear auto reconnect and doesn't need to be set externally, also do we need to be explicitly shutting down subsystem here?  What if the JSDebugger destructor calls ```Shutdown```, see note in destructor.